cfgov/jinja2/v1/owning-a-home/_templates/form-explainer.html

```diff
@@ -0,0 +1,235 @@
+
+{# ==========================================================================
+
+   render( data )
```

Comment:
TRIVIAL: it would be good to normalize whether there are spaces in the parameter in the macros in this PR. Both styles are present.

Reply:
Fixed.